src/js/track.js

```diff
@@ -0,0 +1,83 @@
+var _ = require('underscore');
+var d3 = require('d3');
+var $ = require('jquery');
+var D3SVGCellRenderer = require('./d3_svg_cell_renderer');
+var utils = require('./utils');
+
+module.exports = {};
+
+var defaultTrackConfig = {
+	label: 'Gene',
+	id_member: 'sample',
```

Comment:
I dunno, `column_identifier`? The word `member` isn't use elsewhere in the code to reference elements.